schema/swagger.yml

```diff
@@ -1175,7 +1175,7 @@ definitions:
         type: string
         format: signature
         example: 2821d93a742c4edf5fd960efad41a4def7bf0fd0f7c09869aed524f6f52bf9c97a617095e2c712bd28b4279078a29509b339ac55187854006591aa759784c205
-      multisignatures:
```

Comment:
@diego-G If this change is aligned with LiskJS then its good to go.

Reply:
@willclarktech @shuse2 please if you aren't yet, be aware of it

Reply:
@diego-G Can you summarise the change we need to implement?

Reply:
As we discussed, there you go @willclarktech https://github.com/LiskHQ/lisk-js/issues/640